core/tauri/src/api/process/command.rs

```diff
@@ -261,9 +261,17 @@ impl Command {
     let guard_ = guard.clone();
     spawn(move || {
       let _lock = guard_.read().unwrap();
-      let reader = BufReader::new(stdout_reader);
-      for line in reader.lines() {
+      let mut reader = BufReader::new(stdout_reader);
+
+      let mut buf = Vec::new();
+      loop {
+        buf.clear();
+        let n = read_command_output(&mut reader, &mut buf).unwrap();
```

Comment:
is there a reason to crash on error here instead of just using `?` to bubble it up?

Reply:
fixed it